modules/acknowledgements/jsx/Table.js

```diff
@@ -0,0 +1,29 @@
+class Table extends React.Component {
```

Comment:
You should be using `StaticDataTable` or `DynamicDataTable` components from `/jsx` folder instead of reinventing the wheel. If some functionality it lacking, it can always be added on the side